File: client/src/app/_dtos/admin/createSelDecisionDto.ts
import { CreateSelDecision } from "../../_models/admin/createSelDecision";


export class createSelDecisionDto
{
	cVRefId: number=0;
	decisionDate: Date=new Date();
	selectionStatus: string='';
	rejectionReason: string = '';
	remarks: string='';
}

export class selDecisionsToAddParams
{
	selDecisionsToAddDto: CreateSelDecision[]=[];
	advisesToClients: boolean=false;
	selectionEmailToCandidates: boolean = false;
	rejectionEmaiLToCandidates: boolean=false;
	rejectionSMSToCandidates: boolean = false;
	selectionSMSToCandidates: boolean = false;
  
}

File: api/DTOs/Admin/CreateSelDecisionDto.cs
namespace api.DTOs.Admin
{
    public class CreateSelDecisionDto
    {
        public int CVRefId { get; set; }
        public DateTime DecisionDate { get; set; }
        public string SelectionStatus { get; set; }
        //public string RejectionReason { get; set; }
        public string Remarks { get; set; }
    }
}